src/main/scala/ai/privado/exporter/CollectionExporter.scala

```diff
@@ -115,6 +141,33 @@ class CollectionExporter(cpg: Cpg) {
     )
   }
 
+  def processByLocalVariableId(
```

Comment:
@hiteshbedre We don't need this additional function and also `getCollectionUrlForLocalVar` the existing one can be used by a minor update